examples/tic-tac-toe/cli/src/turn_cap.rs

```diff
@@ -2,12 +2,11 @@
 // Modifications Copyright (c) 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use serde::Deserialize;
 use iota_types::base_types::ObjectID;
+use serde::Deserialize;
```

Comment:
Wait what's happening here? Why is the format changing? Are we not checking format of examples?

Reply:
apparently not, because when I save `game.rs` it also changes the format

Reply:
https://github.com/iotaledger/iota/issues/4480